Dockerfile

```diff
@@ -1,2 +1,2 @@
-FROM rust:1.69.0
+FROM rustlang/rust:nightly
```

Comment:
It's important that `reaper-rs` can build with stable Rust, and ideally also the "generate" step. What nightly feature are you using?